src/screens/atomic-swap/ProposalList.js

```diff
@@ -12,14 +12,18 @@ import { useDispatch, useSelector } from "react-redux";
 import { useHistory } from 'react-router-dom';
 import Loading from "../../components/Loading";
 import { proposalFetchRequested, proposalRemoved } from "../../actions";
-import { PROPOSAL_DOWNLOAD_STATUS, updatePersistentStorage } from "../../utils/atomicSwap";
+import {
+    PROPOSAL_DOWNLOAD_STATUS,
+    updatePersistentStorage
+} from "../../utils/atomicSwap";
 import walletUtil from "../../utils/wallet";
 import { GlobalModalContext, MODAL_TYPES } from '../../components/GlobalModal';
 
 export default function ProposalList (props) {
     const history = useHistory();
     const dispatch = useDispatch();
     const modalContext = useContext(GlobalModalContext);
+    const wallet = useSelector(state => state.wallet);
```

Comment:
Why do you need it? I didn't see you using it.

Reply:
I was not needed anymore, removed on [081005e](https://github.com/HathorNetwork/hathor-wallet/pull/400/commits/081005edfb48b8746d565a0361bb41382b098567)